[DO NOT MERGE] Community Interview (#8837)
GoogleChrome/web.dev: src/site/content/en/blog/community-highlight-shuyi/index.md

---
layout: post
title: 'Community highlight: Olutimilehin Olushuyi'
authors: 
  - alexandrawhite
hero: image/VbsHyyQopiec0718rMq2kTE1hke2/8nibxhPpYh7CcJmZubSa.png
alt: 'Learn Accessibility! Community highlight.'
thumbnail: image/VbsHyyQopiec0718rMq2kTE1hke2/8nibxhPpYh7CcJmZubSa.png

> subhead: >
>   Olutimilehin Olushuyi is a lawyer, new to accessibility. We talked about his battle with JavaScript, international standards, and the importance of reading a website's contents.
> description: >
>   One of a series of interviews with people from the web development community who are workinf in accessibility. This time I spoke with Olutimilehin Olushuyi.

Author: ```suggestion
subhead: >
  Olutimilehin Olushuyi is a lawyer, new to accessibility. We talked about his battle with JavaScript,
  international standards, and the importance of reading a website's contents.
description: >
  One of a series of interviews with people from the web development community who
  work in web accessibility. This time I spoke with Olutimilehin Olushuyi.
```

> date: 2022-10-31

Author: ```suggestion
date: 2022-11-10
```

---
layout: post
title: 'Community highlight: Olutimilehin Olushuyi'
authors: 
  - alexandrawhite
hero: image/VbsHyyQopiec0718rMq2kTE1hke2/8nibxhPpYh7CcJmZubSa.png
alt: 'Learn Accessibility! Community highlight.'
thumbnail: image/VbsHyyQopiec0718rMq2kTE1hke2/8nibxhPpYh7CcJmZubSa.png
subhead: >
  Olutimilehin Olushuyi is a lawyer, new to accessibility. We talked about his battle with JavaScript,
  international standards, and the importance of reading a website's contents.
description: >
  One of a series of interviews with people from the web development community who
  work in web accessibility. This time I spoke with Olutimilehin Olushuyi.

> date: 2022-11-10

Author: ```suggestion
date: 2022-11-09
```
